article/app/views/fragments/recipeArticleBody.scala.html

```diff
@@ -93,9 +107,20 @@
 
 }
 
-
 @recipePage(recipe: model.content.RecipeAtom, i: Int) = {
     <div data-recipe-index="recipe__@i" id="@recipe.atom.id" class="js-recipe__article--structured recipe__article--structured" itemtype="http://schema.org/Recipe">
+        <div class="recipe__article--structured-image mobile-only">
+        @RecipeAtom.picture(recipe).orElse(maybeMainImage).map { image =>
+            @fragments.image(
+                picture = image,
+                classes = Seq("recipe__image-inline mobile-only"),
+                imageAltText = "",
+                widths = MainMedia.inline,
```

Comment:
Are [inline](https://github.com/guardian/frontend/blob/master/common/app/layout/WidthsByBreakpoint.scala#L152) the right widths for this? Might have to create your own if they are not.